src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.HandleDictionary.cs

```diff
@@ -166,7 +166,7 @@ private static void WriteExtensionData(Utf8JsonWriter writer, ref WriteStackFram
                 Debug.Assert(entry.Key is string);
 
                 string propertyName = (string)entry.Key;
-                element.WriteAsProperty(propertyName.AsSpan(), writer);
+                element.WriteAsProperty(propertyName, writer);
```

Comment:
@steveharter, updated this from https://github.com/dotnet/corefx/pull/37690#discussion_r284810711